Makefile

```diff
@@ -63,6 +63,10 @@ lint: vendor golangci-lint-config.yaml
 	golangci-lint run --config golangci-lint-config.yaml
 .PHONY: lint
 
+tools: vendor
+	@go install github.com/golangci/golangci-lint/cmd/golangci-lint
```

Comment:
Do you need to do this since you are adding it as an import in the tools package?

Reply:
yes, either this is needed or I will have to prefix 'golangci-lint' with 'go run github.com/golangci'

Reply:
But after you `go build tools`, because you imported golangci-lint, does that end up with the binary in `go env GOPATH + "/bin"` ?

Reply:
yes,